Dockerfile

```
@@ -1,6 +1,11 @@
 FROM centos:5
 MAINTAINER Remi Hakim @remh
 
+COPY ./CentOS-Base.repo /etc/yum.repos.d/CentOS-Base.repo
+COPY ./libselinux.repo /etc/yum.repos.d/libselinux.repo
+
+RUN yum -y update
+
 RUN yum -y install \
```

`update` and `install` [should always be in the same layer](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run)

```suggestion
RUN yum -y update && \
    yum -y install \
```